util/dvsim/Deploy.py

```diff
@@ -438,6 +441,7 @@ def _set_attrs(self):
         self.build_mode = self.name
         self.job_name += f"_{self.build_mode}"
         self.fail_patterns = self.build_fail_patterns
+        self.fail_patterns = self.build_fail_patterns
```

Comment:
s/fail/pass?

Reply:
That should be pass! Thanks for catching that, somehow did not push the latest change..